docs/user/README.md

```diff
@@ -0,0 +1,217 @@
+# ModelarDB installation and usage
+This document describes how to set up and use ModelarDB. Installation instructions are provided for 
+Linux, macOS, FreeBSD, and Windows. Once installed, using ModelarDB is consistent across all platforms.
+
+ModelarDB intentionally does not gather usage data. So, all users are highly encouraged to post comments, suggestions, 
```

Comment:
Maybe move this to the introduction in the README as I would consider it a selling point that we do not collect telemetry and we want to encourage users to provide us with feedback.

Reply:
Fixed.